### PR TITLE
Release/1.1.1

### PR DESCRIPTION
**Pull request by sphuber**

### CODE REVIEW

**Comment by sphuber** [comment]
@ltalirz note that for the last release you said we should still go through a separate PR to just make the release commit, but looking at this more closely now I really don't think it is necessary. The reviewer just has to check the last commit, to see that all release steps are correctly included. Do you agree?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
...
+- Validate label string at code setup stage [[#3793]](https://github.com/aiidateam/aiida-core/pull/3793)
+- Reuse `prepend_text` and `append_text` in `verdi computer/code duplicate` [[#3788]](https://github.com/aiidateam/aiida-core/pull/3788)
+- Fix broken imports of `urllib` in various locations [[#3767]](https://github.com/aiidateam/aiida-core/pull/3767)
+- Match headers with actual output for `verdi data structure list` [[#3756]](https://github.com/aiidateam/aiida-core/pull/3756)
+- Disable caching for the `Data` node subclass. Note that this should not affect the behavior [[#3807]](https://github.com/aiidateam/aiida-core/pull/3807)
+
+
 ## v1.1.0
 
 **Note Bene:** although this is a minor version release, the support for python 2 is dropped [(#3566)](https://github.com/aiidateam/aiida-core/pull/3566) following the reasoning outlined in the corresponding [AEP001](https://github.com/aiidateam/AEP/tree/master/001_drop_python2).
```
Comment (ltalirz) [inline CHANGELOG.md]:
```suggestion
**Nota Bene:** although this is a minor version release, the support for python 2 is dropped [(#3566)](https://github.com/aiidateam/aiida-core/pull/3566) following the reasoning outlined in the corresponding [AEP001](https://github.com/aiidateam/AEP/tree/master/001_drop_python2).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## v1.1.1
+
+### Changes
+- Emit a warning when input port specifies a node instance as default [[#3466]](https://github.com/aiidateam/aiida-core/pull/3466)
+- `BaseRestartWorkChain`: require process handlers to be instance methods [[#3782]](https://github.com/aiidateam/aiida-core/pull/3782)
+- `BaseRestartWorkChain`: add method to enable/disable process handlers [[#3786]](https://github.com/aiidateam/aiida-core/pull/3786)
+- Remove conda activation from configure-aiida.sh script [[#3791]](https://github.com/aiidateam/aiida-core/pull/3791)
```
Comment (ltalirz) [inline CHANGELOG.md]:
```suggestion
- Docker container: Remove conda activation from configure-aiida.sh script [[#3791]](https://github.com/aiidateam/aiida-core/pull/3791)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+- Add fixtures to clear the database before or after tests [[#3783]](https://github.com/aiidateam/aiida-core/pull/3783)
+- `verdi status`: add the configuration directory path to the output [[#3587]](https://github.com/aiidateam/aiida-core/pull/3587)
+- `QueryBuilder`: add support for `datetime.date` objects in filters [[#3796]](https://github.com/aiidateam/aiida-core/pull/3796)
+
+### Bug fixes
+- Write migrated config to disk in `Config.from_file` [[#3797]](https://github.com/aiidateam/aiida-core/pull/3797)
+- Caching: fix configuration spec and validation [[#3785]](https://github.com/aiidateam/aiida-core/pull/3785)
+- Fix bugs in `Node._store_from_cache` and `Node.repository.erase` [[#3777]](https://github.com/aiidateam/aiida-core/pull/3777)
```
Comment (ltalirz) [inline CHANGELOG.md]: I'd probably put this in first place - this is a very important bug fix
```suggestion
- Fix bugs in `Node._store_from_cache` and `Node.repository.erase` that could result in calculations not being reused [[#3777]](https://github.com/aiidateam/aiida-core/pull/3777)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,27 @@
...
+- Validate label string at code setup stage [[#3793]](https://github.com/aiidateam/aiida-core/pull/3793)
+- Reuse `prepend_text` and `append_text` in `verdi computer/code duplicate` [[#3788]](https://github.com/aiidateam/aiida-core/pull/3788)
+- Fix broken imports of `urllib` in various locations [[#3767]](https://github.com/aiidateam/aiida-core/pull/3767)
```
Comment (ltalirz) [inline CHANGELOG.md]:
```suggestion
- Fix broken imports of `urllib` in various locations including `verdi import` [[#3767]](https://github.com/aiidateam/aiida-core/pull/3767)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,27 @@
...
+- Match headers with actual output for `verdi data structure list` [[#3756]](https://github.com/aiidateam/aiida-core/pull/3756)
+- Disable caching for the `Data` node subclass. Note that this should not affect the behavior [[#3807]](https://github.com/aiidateam/aiida-core/pull/3807)
```
Comment (ltalirz) [inline CHANGELOG.md]:
```suggestion
- Disable caching for the `Data` node subclass (this should not affect usual caching behavior). [[#3807]](https://github.com/aiidateam/aiida-core/pull/3807)
```

**Comment by sphuber** [comment]
@ltalirz thanks for the suggestions, I applied all of them

**Pull request merged**